app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.js

```diff
@@ -1,113 +1,114 @@
-;(function (global) {
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.analyticsGA4 = window.GOVUK.analyticsGA4 || {}
+window.GOVUK.analyticsGA4.analyticsModules = window.GOVUK.analyticsGA4.analyticsModules || {};
+
+(function (analyticsModules) {
   'use strict'
 
-  var GOVUK = global.GOVUK || {}
-  GOVUK.analyticsGA4 = GOVUK.analyticsGA4 || {}
-
-  GOVUK.analyticsGA4.pageViewTracker = {
-    PIIRemover: new GOVUK.analyticsGA4.PIIRemover(), // imported in analytics-ga4.js
-    nullValue: null,
-
-    sendPageView: function () {
-      if (window.dataLayer) {
-        var data = {
-          event: 'page_view',
-          page_view: {
-            location: this.getLocation(),
-            referrer: this.getReferrer(),
-            title: this.getTitle(),
-            status_code: this.getStatusCode(),
-
-            document_type: this.getMetaContent('format'),
-            publishing_app: this.getMetaContent('publishing-app'),
-            rendering_app: this.getMetaContent('rendering-app'),
-            schema_name: this.getMetaContent('schema-name'),
-            content_id: this.getMetaContent('content-id'),
-
-            section: this.getMetaContent('section'),
-            taxon_slug: this.getMetaContent('taxon-slug'),
-            taxon_id: this.getMetaContent('taxon-id'),
-            themes: this.getMetaContent('themes'),
-            taxon_slugs: this.getMetaContent('taxon-slugs'),
-            taxon_ids: this.getMetaContent('taxon-ids'),
-
-            language: this.getLanguage(),
-            history: this.getHistory(),
-            withdrawn: this.getWithDrawn(),
-            first_published_at: this.stripTimeFrom(this.getMetaContent('first-published-at')),
-            updated_at: this.stripTimeFrom(this.getMetaContent('updated-at')),
-            public_updated_at: this.stripTimeFrom(this.getMetaContent('public-updated-at')),
-            publishing_government: this.getMetaContent('publishing-government'),
-            political_status: this.getMetaContent('political-status'),
-            primary_publishing_organisation: this.getMetaContent('primary-publishing-organisation'),
-            organisations: this.getMetaContent('analytics:organisations'),
-            world_locations: this.getMetaContent('analytics:world-locations')
-          }
+  var PageViewTracker = function () {}
```

Comment:
Would there be any mileage in turning this into an object rather than a function?